pennylane/templates/subroutines/commuting_evolution.py

```diff
@@ -79,7 +79,8 @@ class CommutingEvolution(Operation):
     .. details::
         :title: Usage Details
 
-        The template is used inside a qnode:
+        The template is used inside a qnode.

```

Comment:
have you intentionally modified this file?

Reply:
ops! I think yes, I roll back the change